### PR TITLE
Refs [TASK][25319] [Admin][Department] Add department

### PR DESCRIPTION
**Pull request by duytu131995**
https://edu-redmine.sun-asterisk.vn/issues/25319
Screenshot from 2020-07-07 11-16-55

### CODE REVIEW

**Comment by duytu131995** [comment]
ready

File: app/controllers/admin/departments_controller.rb [hunk 1]
```diff
@@ -2,4 +2,27 @@ class Admin::DepartmentsController < AdminController
...
+
+  def new
+    @department = Department.new
+  end
+
+  def create
+    @department = Department.new department_params
+    if @department.save
+      flash[:success] = t "admin.departments.create.success"
+      redirect_to admin_department_path id: @department
```
Comment (ngocvt-0484) [inline app/controllers/admin/departments_controller.rb:16]: cần id k e

Comment (duytu131995) [reply app/controllers/admin/departments_controller.rb:16]: Dạ không chị ạ :))

**Comment by duytu131995** [comment]
ready

**Pull request merged**